app/jobs/appointment_reminder_job.rb

```diff
@@ -4,8 +4,7 @@ class AppointmentReminderJob < ApplicationJob
   def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
-      owner: appointment.guider,
-      message: 'sent a reminder'
```

Comment:
Oh disregard that I see what you mean. Message is defaulting to '' anyway so no reason for that.